share/github-backup-utils/ghe-restore-mysql

```diff
@@ -28,15 +28,22 @@ ghe_remote_version_required "$GHE_HOSTNAME"
 # The directory holding the snapshot to restore
 snapshot_dir="$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT"
 
-# Check if the backup is binary by looking up the sentinel file 
+# Check if the backup is binary by looking up the sentinel file
 is_binary_backup(){
-  test -f $snapshot_dir/mysql-binary-backup-sentinel 
+  test -f $snapshot_dir/mysql-binary-backup-sentinel
 }
 
-# if mysql.backup.binary feature flag is on
-is_binary_backup_feature_on(){
-  ghe-ssh "$GHE_HOSTNAME" ghe-config --true "mysql.backup.binary"
-}
+if is_binary_backup_feature_on; then
+  echo "Appliance is configured for binary backups."
+else
+  echo "Appliance is configured for logical backups."
+fi
+
+if is_binary_backup; then
```

Comment:
is that possible:
The Appliance is configured for binary backups, but Backup being restored is a logical backup?

Reply:
Looks like it: https://github.com/github/backup-utils/blob/echo-msyql-backup-strategy/share/github-backup-utils/ghe-restore-mysql#L78-L81